package.json

```diff
@@ -1,93 +1,34 @@
 {
-  "name": "iframe-coordinator",
-  "version": "4.5.2",
-  "description": "Tools for coordinating embedded apps via iframes.",
-  "dependencies": {
-    "cheerio": "^1.0.0-rc.10",
-    "commander": "^2.20.3",
-    "decoders": "1.15.0",
-    "dev-cert-authority": "^1.1.1",
-    "express": "^4.17.3",
-    "find-root": "^1.1.0",
-    "http-proxy-middleware": "^1.3.1"
-  },
-  "files": [
-    "dist/client.js",
-    "dist/host.js",
-    "dist/index.js",
-    "dist/es5/client.js",
-    "dist/es5/host.js",
-    "dist/es5/index.js",
-    "dist/**/*.d.ts",
-    "cli/ifc-cli.js",
-    "cli/example-ifc.config.js",
-    "cli/embedded-app/dist/**/*"
+  "name": "iframe-coordinator-monorepo",
+  "version": "5.0.0",
+  "workspaces": [
+    "./packages/iframe-coordinator",
+    "./packages/ifc-cli",
+    "./apps/*"
   ],
-  "main": "dist/index.js",
-  "module": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "bin": {
-    "ifc-cli": "cli/ifc-cli.js"
+  "scripts": {
+    "build": "npm run build --workspaces --if-present",
+    "start-client-example": "npm run start --workspace=ifc-example-client",
+    "test": "npm test --workspaces --if-present",
+    "commit": "git-cz",
+    "lint": "commitlint -f de759fc509cf39895450dc31742cfb36b08e01b8",
+    "update-version": "./scripts/update-version.mjs",
+    "release": "npm run release --workspace=iframe-coordinator --workspace=ifc-cli",
+    "publish": "npm publish --workspace=iframe-coordinator --workspace=ifc-cli",
```

Comment:
I think we'll need to change the `npm publish` command in the Jenkinsfile to be `npm run publish` rather than `npm publish` for this to work. I might even rename this to `publish-libs` just to avoid confusion of this script with the `npm publish` command.